design-documents/graph-ql/coverage/customer/Wishlist.graphqls

```diff
@@ -33,8 +56,9 @@ type WishlistItems {
 }
 
 input WishlistItemUpdateInput {
-    wishlist_item_id: ID
```

Comment:
we can't remove any of the existing fields, but deprecate them and also add wishlist_item_uid